test/e2e/arktos/multi_tenancy/test_suites/tenant_init_delete_test.yaml

```diff
@@ -15,14 +16,15 @@ Variables:
 # check the system tenant is created automatically and cannot be deleted
 ############################################################################################################
 Tests:
-  - Command: ${kubectl} create tenant system
+  - BeforeTestMessage: Verifying the system tenant is created automatically and cannot be deleted ...
+    Command: ${kubectl} create tenant system
     ShouldFail: true
     OutputShouldContain: 
-    - "Error from server (AlreadyExists): tenants \"system\" already exists"
+    - "Error from server (AlreadyExists): tenants \"system\" already exists\n"
     
   - Command: ${kubectl} get tenants -o json | jq -r '.items[] | [.metadata.name, .status.phase] | @tsv'
     OutputShouldContain: 
-    - "system\tActive"
```

Comment:
how to choose between \t and space?

Reply:
No difference, actually. I changed from "\t" to a real tab space, because I think it is more friendly to readers' eyes. :-)